pkgs/servers/monitoring/prometheus/xmpp-alerts.nix

```diff
@@ -2,16 +2,16 @@
 
 pythonPackages.buildPythonApplication rec {
   pname = "prometheus-xmpp-alerts";
-  version = "0.4.2";
+  version = "0.4.2+git";
```

Comment:
From the traceback this looks alot like an upstream issue, so the proper course of action would be to report it upstream and have them make a new release.

In the meantime we could apply a patch to fix the situation if you can identify one.